test/README.md

```diff
@@ -1,6 +1,6 @@
 # Self-hosted tests
 
-This directory contains tests that are run using a stable version of AVA. You can run them using `npx test-ava`.
+This directory contains tests that are run using a stable version of AVA. You can run them using `npx test-ava` at the project's root.
```

Comment:
Maybe I am not that familiar with the project yet, but I struggled a bit to understand why `npx test-ava` wasn't working until I tried it at the project's root.

Reply:
It's because there's a `node_modules` inside `test/` as well. Maybe we should symlink `test-ava` from there.

Reply:
Sounds good, will give it a try.

Reply:
Symlink added